Signal/src/util/DateUtil.m

```diff
@@ -78,11 +78,8 @@ + (BOOL)dateIsOlderThanToday:(NSDate *)date
 
 + (BOOL)dateIsOlderThanToday:(NSDate *)date now:(NSDate *)now
 {
-    NSCalendar *calendar = [NSCalendar currentCalendar];
-
-    NSUInteger dateDayOfEra = [calendar ordinalityOfUnit:NSCalendarUnitDay inUnit:NSCalendarUnitEra forDate:date];
```

Comment:
`[NSCalendar  ordinalityOfUnit: inUnit:]` ignores timezone when the ordinality is "era".